lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/GroupCommClient.cs

```diff
@@ -46,17 +44,17 @@ public class GroupCommClient : IGroupCommClient
         /// </summary>
         /// <param name="groupConfigs">The set of serialized Group Communication configurations</param>
         /// <param name="taskId">The identifier for this task</param>
-        /// <param name="groupCommNetworkObserver">The network handler to receive incoming messages
         /// for this task</param>
         /// <param name="networkService">The network service used to send messages</param>
         /// <param name="configSerializer">Used to deserialize Group Communication configuration</param>
+        /// <param name="injector">injector forked from the injector that creates this instance</param>
         [Inject]
         public GroupCommClient(
```

Comment:
We should make these constructors `private`, as they are only called by Tang. I'm fine with doing this breaking change now, as the group communications library is all new anyhow.